### ISSUE TITLE
Plugin does't allow to change masked input value while handling change event

### ISSUE THREAD

**Issue by rommcr** [RobinHerbots/Inputmask#807]
Example:

```
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
<script src="jquery.inputmask.bundle.js"></script>

<input type="text" id="t" data-inputmask="'alias' : 'decimal'" value="2">
<input type="button" id="b" value="Test">

<script>
$(function(){
    $("#t").inputmask();
    $("#t").change(function(){
        $(this).val(50);
    });
    $("#b").click(function(){
        alert($("#t").val());
    })

})
</script>
```

When handling .change event on masked input field and trying to set value from event handler, inputmask plugin keeps old value until mouseover over the text element. Only 

How to:
1. change value in input
2. fire onchange event ( value still like nothing happend )
3. mouseover over the texfield ( value changed to "50" )

window.setTimeout(function(){ $("#t").val(25) }, 0)

can helps, but that's not solution, isn't it?

**Comment by RobinHerbots** [comment]
@rommcr , 

Yes I know, I will have a look.

**Comment by RobinHerbots** [comment]
@rommcr , 

It is fixed in the latest version.

**Comment by kaarthikraajan** [comment]
I am facing a similar issue to reset the value with val(''), because on focus or mouse hover I still see the old value in the input mask field.

May i know the version of this fix?

**Comment by RobinHerbots** [comment]
@kaarthikraajan , 

DId you try with version 3.1.62

**Comment by kaarthikraajan** [comment]
Great Robin!! I just tried the latest version and it seem to be working fine, this is my work around for my android backspace issue#867 (I am definitely looking forward to your updated version 3.3)

I have introduces a clearable textbox now in mobile version to address this issue where I was using the val() to clear the input masked field, now when I tried with your latest 3.1.63 I am able to see val() resetting the value sucessfuly. I will continue testing and will let you know in case of any issues.

Thanks again for your response!!
